main.py

```diff
@@ -15,12 +15,25 @@
 def get_data(metrics: list, tickers: list):
     fetcher = data_fetcher.DataFetcher("downloaded_data")
     fetcher.init_downloads_directory()
+    failed_list = []
+    cnt = 0
```

Comment:
The pythonic way to use counter in a for loop is to use enumerate()

Reply:
nice one! I will use it from now on